metricbeat/module/kafka/consumergroup/_meta/data.json

```diff
@@ -1,36 +1,44 @@
 {
-    "@timestamp":"2016-05-23T08:05:34.853Z",
-    "type":"metricsets",
-    "beat":{
-        "hostname":"localhost",
-        "name":"localhost",
-        "version": "6.0.0-alpha1"
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
     },
-    "metricset":{
-        "host":"localhost",
-        "module":"kafka",
-        "name":"consumergroup",
-        "rtt":269
-    },
-    "kafka":{
-        "consumergroup":{
-            "id": "group",
-            "topic": "test",
-            "partition": 0,
-            "client": {
-              "host": "127.0.0.1",
-              "id": "client0",
-              "member_id": "client0-d20b677a-5740-433e-a7f8-fbdab1f0f150"
-            },
+    "kafka": {
+        "broker": {
+            "address": "172.18.0.2:9092",
+            "id": 0
+        },
+        "consumergroup": {
             "broker": {
-                "address": "kafka0:9092",
+                "address": "172.18.0.2:9092",
```

Comment:
Interesting, in my laptop the broker address is different for consumergroup and partition metricsets, this is not good for correlations :thinking:

Reply:
Using advertised address in both metricsets now, so it can be more easily used for correlations.

Reply:
Yes, the problem was that in one metricset we were using the address we use to connect, and on the other the advertised address, so the broker was the same, but different addresses were being collected.